addons/project_todo/views/project_task_views.xml

```diff
@@ -50,17 +48,16 @@
                                 <div class="o_kanban_record_body o_todo_kanban_card_body" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
                                     <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
                                 </div>
-                                <div class="o_kanban_record_bottom">
-                                    <div class="oe_kanban_bottom_left"/>
-                                    <div class="oe_kanban_bottom_right"  t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                        <div t-attf-class="d-flex #{todoHasAssignees ? 'w-100 align-items-center justify-content-end' : 'align-items-end'}">
-                                            <div class="d-flex align-items-center me-2 mt-2">
-                                                <div t-if="todoHasAssignees">
```

Comment:
I you don't use `todoHasAssignees`, also remove its declaration on line 34. I'm still wondering whether it could not be used to avoid showing only one avatar*, see my comment on the task.

_* if we need to only display the avatars if more than one, this variable could be use to set a class used in the stylesheet to hide avatars._

Reply:
@ravb-odoo : Following Alexandra's feedback, it should be nice to also implement the hiding of the avatar in the kanban card in case there is only one assignee. In my opinion, an easy way to do so would be:
1. Keep the variable `todoHasAssignees`
2. Use it to had a class on the avatar field if only 1 assignee
3. In the stylesheet, hide the avatar (`display: none`) if it has this particular class on a parent div. 
Of course if you have another idea to do that, feel free, it's only a suggestion to avoid costly customization.

Reply:
Hello @huca-odoo ,
I have done the required changes.
Thank You:)